modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -37,6 +40,11 @@ class OrtResultAnalyzer : ManualAnalyzer() {
 
     override fun getName() = "OrtResult"
 
+    override fun configure(configMap: Map<String, String>?) {
+        super.configure(configMap)
+        omitExcludedArtifacts = ! getBooleanConfigValue(keepExcludedArtifactsProperty, configMap)
```

Comment:
Nit: I'm not sure about the style for Antenna, but I'd expect to be no space after `!`.